fluentd-gcp-scaler/scaler.sh

```diff
@@ -23,7 +23,7 @@ log() {
 apply_scaling() {
   # This is assuming there is a ScalingPolicy installed in the cluster.
   # See https://github.com/justinsb/scaler for more details.
-  if ! kubectl get scalingpolicies -n ${NAMESPACE} ${SCALING_POLICY} 2> /dev/null
+  if ! kubectl get scalingpolicies -n ${NAMESPACE} ${SCALING_POLICY} &> /dev/null
```

Comment:
This is `/bin/sh`, not Bash. I don't think we can use `&>` here, using `>/dev/null 2>&1` is probably safer.

Reply:
Good point, I've changed it.